<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TroopPageConditionDialog</class>
 <widget class="QWidget" name="TroopPageConditionDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YaRXP - Troop Page Conditions</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="3">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>is ON</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="spin_actor">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSpinBox" name="spin_turn_a">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spin_turn_b">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="check_enemy">
       <property name="text">
        <string>Enemy</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QCheckBox" name="check_switch">
       <property name="text">
        <string>Switch</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="check_turn">
       <property name="text">
        <string>Turn</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>'s HP</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="spin_enemy">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>'s HP</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="check_actor">
       <property name="text">
        <string>Actor</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>% or below</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QComboBox" name="combo_enemy">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QComboBox" name="combo_actor">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>% or below</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="2">
      <widget class="SwitchVariableWidget" name="widget_switch" native="true">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_ok">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SwitchVariableWidget</class>
   <extends>QWidget</extends>
   <header>dialogs/switchvariablewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>check_turn</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_turn_a</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_turn</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_turn_b</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_enemy</sender>
   <signal>toggled(bool)</signal>
   <receiver>combo_enemy</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_enemy</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_enemy</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_actor</sender>
   <signal>toggled(bool)</signal>
   <receiver>combo_actor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_actor</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_actor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>128</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_switch</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget_switch</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
